plugins/Scheduler/test.py

```diff
@@ -74,6 +74,13 @@ def test00RemoveZero(self):
         timeFastForward(5)
         self.assertNoResponse(' ', timeout=1)
 
+    def testRemind(self):
+        self.assertNotError('scheduler remind 5 testRemind')
+        timeFastForward(3)
+        self.assertNoResponse(' ', timeout=1)
+        timeFastForward(2)
+        self.assertResponse(' ', 'Reminder: testRemind')
```

Comment:
you should re-add the `timeFastForward(5)` + `assertNoResponse` at the end to check it's not repeated.